src/templates/user/purchases.html

```diff
@@ -3,7 +3,10 @@
 
 <section class="main">
   <div class="subheader">
```

Comment:
Not sure about subheader since it seems a bit marketplacey. The app list is a good candidate though I'd rather get though the krupa bugs on dev first.